ci-operator/config/openshift-knative/serverless-operator/openshift-knative-serverless-operator-main__serverless-p2p-rosa-classic.yaml

```diff
@@ -0,0 +1,187 @@
+    name: "4.17"
+    namespace: origin
+    tag: operator-registry
+  rosa-aws-cli:
+    name: rosa-aws-cli
+    namespace: ci
+    tag: latest
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+images:
+- dockerfile_path: knative-operator/Dockerfile
+  inputs:
+    ocp_ubi-minimal_8:
+      as:
+      - registry.ci.openshift.org/ocp/ubi-minimal:8
+    openshift_release_rhel-9-release-golang-1.22-openshift-4.17:
+      as:
+      - registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.22-openshift-4.17
+  to: serverless-knative-operator
+- dockerfile_path: olm-catalog/serverless-operator/Dockerfile
+  to: serverless-bundle
+- dockerfile_path: olm-catalog/serverless-operator/index/Dockerfile
+  inputs:
+    origin_4.17_operator-registry:
+      as:
+      - registry.ci.openshift.org/origin/4.17:operator-registry
+  to: serverless-index
+- dockerfile_path: openshift-knative-operator/Dockerfile
+  inputs:
+    ocp_ubi-minimal_8:
+      as:
+      - registry.ci.openshift.org/ocp/ubi-minimal:8
+    openshift_release_rhel-9-release-golang-1.22-openshift-4.17:
+      as:
+      - registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.22-openshift-4.17
+  to: serverless-openshift-knative-operator
+- dockerfile_path: openshift/ci-operator/source-image/Dockerfile
+  from: src
+  inputs:
+    ocp_4.17_cli:
+      as:
+      - registry.ci.openshift.org/ocp/4.17:cli
+    openshift_eventing-istio-src_knative-v1.14:
+      as:
+      - registry.ci.openshift.org/openshift/eventing-istio-src:knative-v1.14
+    openshift_eventing-kafka-broker-src_knative-v1.14:
+      as:
+      - registry.ci.openshift.org/openshift/eventing-kafka-broker-src:knative-v1.14
+    openshift_knative-eventing-src_knative-v1.14:
+      as:
+      - registry.ci.openshift.org/openshift/knative-eventing-src:knative-v1.14
+    openshift_knative-serving-src_knative-v1.14:
+      as:
+      - registry.ci.openshift.org/openshift/knative-serving-src:knative-v1.14
+  to: serverless-source-image
+- dockerfile_path: serving/ingress/Dockerfile
+  inputs:
+    ocp_ubi-minimal_8:
+      as:
+      - registry.ci.openshift.org/ocp/ubi-minimal:8
+    openshift_release_rhel-9-release-golang-1.22-openshift-4.17:
+      as:
+      - registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.22-openshift-4.17
+  to: serverless-ingress
+- dockerfile_path: serving/metadata-webhook/Dockerfile
+  inputs:
+    ocp_ubi-minimal_8:
+      as:
+      - registry.ci.openshift.org/ocp/ubi-minimal:8
+    openshift_release_rhel-9-release-golang-1.22-openshift-4.17:
+      as:
+      - registry.ci.openshift.org/openshift/release:rhel-9-release-golang-1.22-openshift-4.17
+  to: serverless-metadata-webhook
+releases:
+  latest:
+    release:
+      channel: candidate
+      version: "4.17"
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: 417-rc-iib
```

Comment:
Just curious if you want to use the suffix `-iib`. This PR is not using installation from IIB, as far as I know.

Otherwise this PR looks good.

Reply:
@mgencur we use `iib` suffix for the jobs that includes pre-GA operators installation.
And we plan to switch to our IIB install script when IIB for v1.34 will be available.

Reply:
OK. Let's see if IIB for 1.34 is available soon enough before you actually start testing OCP 4.18. Anyway, the name is not very important here.

Reply:
/lgtm